lib/model.php

```diff
@@ -76,10 +80,18 @@ public function purge()
      */
     private function _getStore()
     {
+        // FIXME
+        // Workaround so that config value don't need to be changed
```

Comment:
Well... in this case you just have to change the path in the config to keep it running.